.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: rust
 sudo: false
 rust:
```

Comment:
I think the code depends on these feature:
- `Box::into_raw`
- `Box::from_raw`
- `(*const Trait) as *const Struct` (used in one of the doc tests)

The first two can easily be changed into `mem::transmute`, but I'm not sure about the last one.